magento-research/pwa-studio
# Release notes 3.0

## Description

Update the CHANGELOG file with release notes for 3.0.0.

## Related Issue

An action item of #1239 

## Verification Steps
1. Open CHANGELOG.md
2. Verify 3.0.0 release notes content exists and covers all the work done for 3.0.0.

## Checklist:
